src/storage/src/source/reclock/compat.rs

```diff
@@ -124,17 +133,31 @@ impl PersistHandle {
     ) -> anyhow::Result<Self> {
         let mut persist_clients = persist_clients.lock().await;
         let persist_client = persist_clients
-            .open(metadata.persist_location)
+            .open(metadata.persist_location.clone())
             .await
             .context("error creating persist client")?;
         drop(persist_clients);
 
+        let since_handle: SinceHandle<_, _, _, _, PersistEpoch> = persist_client
+            .open_critical_since(
+                metadata.data_shard.clone(),
+                PersistClient::CONTROLLER_CRITICAL_SINCE,
```

Comment:
Do we want to use `CONTROLLER_CRITICAL_SINCE` or create sth like a `REMAP_CRITICAL_SINCE` just for the remap operator?

In the future, we might then want to have both a critical handle for the controller (to ensure that the remap shard is queryable at the since at which it needs to be queryable) and for the remap operator (to ensure that the remap shard stays at a since such that we can safely restart).

Reply:
@danhhz That last part, of having to critical since handles might be interesting for you, because of your comment on `CONTROLLER_CRITICAL_SINCE`.

Reply:
@aljoscha I would use `REMAP_CRITICAL_SINCE` because we want to save the `CONTROLLER_` one for when the remap shard can have its critical since handle held in the controller (which, as far as I can tell, is blocked on the remap shard work sean is doing!)

Reply:
my advice is if these are two "logically different" holds, they should be different handles (and thus ids). sounds like they are?

Reply:
I think we should use the `CONTROLLER_CRITICAL_SINCE` because that is what we're logically doing, we just don't have the controller code to do it there right now. If we initialize a second handle id we'd need a migration when we start handling the remap handle from the controller

Reply:
> and for the remap operator (to ensure that the remap shard stays at a since such that we can safely restart).

That should be taken care by an appropriate readpolicy in the controller since it will become just another collection, right?

Reply:
Recording an "offline" decisions: we will use `CONTROLLER_CRITICAL_SINCE` for now and likely move that to the controller once it takes (ha!) control of the remap shard. We likely won't have two critical handles.